From b7f8490681d9bfb5c47f250323e369bed1645aac Mon Sep 17 00:00:00 2001
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Fri, 7 Mar 2014 18:37:48 +0100
Subject: [PATCH 08/16] pc: Add RHEL6 e1000 gPXE image

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1394217468-2864-3-git-send-email-ehabkost@redhat.com>
Patchwork-id: 58050
O-Subject: [RHEL7 qemu-kvm PATCH 2/2] pc: Add RHEL6 e1000 gPXE image
Bugzilla: 1073774
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Amos Kong <akong@redhat.com>

Bugzilla: 1073774
Brew scratch build: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=7157997

The fix for bz#997702 didn't have the RHEL6 e1000 ROM. Add the e1000-0x100e.rom
file from gpxe-roms-qemu-0.9.7-6.9.el6.noarch.rpm.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/i386/pc_piix.c             |   4 ++++
 redhat/Makefile.common        |   2 +-
 redhat/qemu-kvm.spec.template |   3 +++
 redhat/rhel6-e1000.rom        | Bin 0 -> 69120 bytes
 4 files changed, 8 insertions(+), 1 deletion(-)
 create mode 100644 redhat/rhel6-e1000.rom

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c             |    4 ++++
 redhat/Makefile.common        |    2 +-
 redhat/qemu-kvm.spec.template |    3 +++
 redhat/rhel6-e1000.rom        |  Bin 0 -> 69120 bytes
 4 files changed, 8 insertions(+), 1 deletions(-)
 create mode 100644 redhat/rhel6-e1000.rom

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 98a5253..6773fff 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -897,6 +897,10 @@ static QEMUMachine pc_machine_rhel700 = {
         .property = "romfile",\
         .value    = "rhel6-rtl8139.rom",\
     },{\
+        .driver   = "e1000",\
+        .property = "romfile",\
+        .value    = "rhel6-e1000.rom",\
+    },{\
         .driver   = "virtio-net-pci",\
         .property = "romfile",\
         .value    = "rhel6-virtio.rom",\
-- 
1.7.1
